ccmbioinfo/lfs
# CARDS-2383: Create importable spreadsheet for the new YVM questionnaire

DRAFT until #1650 is merged (adding support for custom summary screen message).

Added the questionnaire and an initial survey configuration, as well as an extremely basic clinic configuration (just enough to facilitate testing).

To test:
* build this branch
* start with `-P  prems`
* create a visit for the Your Voice Matters (PMH) "clinic"
* obtain the auth token
* as patient, start filling out the YVM survey
  * check that the message on the start screen corresponds to the intro from the [specs PDF](https://phenotips.atlassian.net/browse/CARDS-2383)
  * check that the overall survey is according to [specs PDF](https://phenotips.atlassian.net/browse/CARDS-2383)
  * check that the message on the summary screen corresponds to the ending from the [specs PDF](https://phenotips.atlassian.net/browse/CARDS-2383)
